README.md

```diff
@@ -1,6 +1,6 @@
 ![Google Cloud Platform Logo](https://cloud.google.com/_static/images/gcp-logo.png)
 # Google Cloud Tools for IntelliJ plugin
-[![Build Status](https://cloud-tools-for-java-testing.appspot.com/buildStatus/icon?job=google-cloud-intellij master build orb)](https://cloud-tools-for-java-testing.appspot.com/job/google-cloud-intellij%20master%20build%20orb/) [![codecov](https://codecov.io/gh/GoogleCloudPlatform/google-cloud-intellij/branch/master/graph/badge.svg)](https://codecov.io/gh/GoogleCloudPlatform/google-cloud-intellij)
+[![Build Status](https://cloud-tools-for-java-testing.appspot.com/buildStatus/icon?job=google-cloud-intellij master build orb (IDEA 2016))](https://cloud-tools-for-java-testing.appspot.com/job/google-cloud-intellij%20master%20build%20orb%20(IDEA%202016)/)
```

Comment:
This is what happens when I merge from the IDE instead of the command line...

Let me fix this real quick.